### DISCUSSION TITLE
Can't import React

### DISCUSSION THREAD

**Discussion by risenxxx** [0xF6/mana_lang#55] [Q&A] [answered]
**Describe the bug**
`import React from 'react'` doesn't work.

**To Reproduce**
Try to import React

** Code To Reproduce **
```csharp
import React from 'react'
```

**Expected behavior**
React being imported.

**Platform (please complete the following information):**
 - (required) OS: MacOS 11.4
 - (optional) VSCode Version: 1.57.1

**Additional context**
I want pizza

**Comment by 0xF6** [comment]
Try 
```mana
#use "facebook/react"
```

**Comment by risenxxx** [comment]
@0xF6 I can't find `#` on my keyboard since I don't use `id` attribute on HTML elements, what's the solution?

**Comment by 0xF6** [comment, marked as answer]
> @0xF6 I can't find `#` on my keyboard since I don't use `id` attribute on HTML elements, what's the solution?

Try connect your keyboard to pc, you may have a problem with that, but i believe in you :trollface:

**Reply by risenxxx** [reply]
<img width="208" alt="CleanShot 2021-06-21 at 01 49 09@2x" src="https://user-images.githubusercontent.com/35740512/122687980-f117d200-d232-11eb-963c-91dde217cab0.png">

I don't have such one.

**Reply by 0xF6** [reply]
reinstall arch btw
